### DISCUSSION TITLE
Default Mailbox Per Domain

### DISCUSSION THREAD

**Discussion by kaji-ryoji** [simple-login/app#650] [Feature Requests]
Hi! I'm an extremely happy paid customer.  I was wondering if default mailbox per domain.

For example, I have 3 domains

- alpha.com
- beta.com
- gamma.com

and I have 3 mailboxes respectively

- alpha@gmail.com
- beta@gmail.com
- gamma@gmail.com

Could we add a default mailbox per domain setting, such that the random email address generated with alpha.com will automatically be forwarded to alpha@gmail.com, beta -> beta and gamma -> gamma?

Thanks

**Comment by nguyenkims** [comment]
@kaji-ryoji hey you can already do that by using the below setting:

<img width="840" alt="image" src="https://user-images.githubusercontent.com/3167780/137183057-3e54fab6-0ce7-49ea-8bea-c5b6806124f7.png">

**Reply by kaji-ryoji** [reply]
@nguyenkims Hi, I'm aware of this feature. However, my particular use case is that if I should generate an email address in chrome extension like deadbeef@beta.com, then it should have its default domain set to beta.com. Currently there's only one default domain we can choose from.

**Reply by nguyenkims** [reply]
Oh I see, this setting is indeed for auto created aliases.

**Reply by kaji-ryoji** [reply]
It it possible to add this functionality to the extension someday? Should I add a github issue to track this feature request as well?

**Reply by nguyenkims** [reply]
Yes it'd be best to create a thread on this forum for this feature request in the browser extension :).
